test/passenger_test.rb

```diff
@@ -67,8 +74,96 @@
       end
     end
   end
-
+  
   describe "net_expenditures" do
-    # You add tests for the net_expenditures method
+    before do
+      # TODO: you'll need to add a driver at some point here.
+      @driver = RideShare::Driver.new(
+        id: 7,
+        name: "Merl Glover Driver",
+        vin: "16026202330372367",
+        status: :AVAILABLE,
+        trips: []
+        )
+      @passenger = RideShare::Passenger.new(
+        id: 9,
+        name: "Merl Glover III",
+        phone_number: "1-602-620-2330 x3723",
+        trips: []
+        )
+        
+        @trip1 = RideShare::Trip.new(
+        id: 8,
+        driver: @driver,
+        passenger: @passenger,
+        start_time: Time.new(2016, 8, 8),
+        end_time: Time.new(2016, 8, 9),
+        cost: 10,
+        rating: 5
+        )
+        @passenger.add_trip(@trip1)
+      
+        @trip2 = RideShare::Trip.new(
+        id: 9,
+        driver: @driver,
+        passenger: @passenger,
+        start_time: Time.new(2016, 8, 11),
+        end_time: Time.new(2016, 8, 12),
+        cost: 11,
+        rating: 5
+        )
+        @passenger.add_trip(@trip2)
```

Comment:
Incorrect indentation:

```suggestion
      @driver = RideShare::Driver.new(
        id: 7,
        name: "Merl Glover Driver",
        vin: "16026202330372367",
        status: :AVAILABLE,
        trips: []
      )
      @passenger = RideShare::Passenger.new(
        id: 9,
        name: "Merl Glover III",
        phone_number: "1-602-620-2330 x3723",
        trips: []
      )
        
      @trip1 = RideShare::Trip.new(
        id: 8,
        driver: @driver,
        passenger: @passenger,
        start_time: Time.new(2016, 8, 8),
        end_time: Time.new(2016, 8, 9),
        cost: 10,
        rating: 5
      )
      @passenger.add_trip(@trip1)
      
      @trip2 = RideShare::Trip.new(
        id: 9,
        driver: @driver,
        passenger: @passenger,
        start_time: Time.new(2016, 8, 11),
        end_time: Time.new(2016, 8, 12),
        cost: 11,
        rating: 5
      )
      @passenger.add_trip(@trip2)
```